cmd/res/configuration.toml

```diff
@@ -65,6 +65,9 @@ PublishTopicPrefix = "edgex/events/device" # /<device-profile-name>/<device-name
   AutoReconnect = "true"
   ConnectTimeout = "5" # Seconds
   SkipCertVerify = "false" # Only used if Cert/Key file or Cert/Key PEMblock are specified
+  [MessageQueue.Topics]
+  CommandRequestTopic = "edgex/command/request/#"         # subscribing for inbound command requests
```

Comment:
Actually, this should have the device service name, otherwise it will receive request for all device services.
```
  CommandRequestTopic = "edgex/command/request/device-rest/#"         # subscribing for inbound command requests
```